clowder.md

```diff
@@ -1,46 +1,102 @@
+
+1. Make sure MongoDB is running  
+
+    cd _mongo installation directory_  
+    ./bin/mongod.exe
+    
+2. Start Clowder frontend  
+
+    cd _clowder directory_  
+    ./sbt run
+    
+3. Set configuration in Clowder directory  
+
+    **/conf/application.conf** includes basic configuration entries. To override these, create a file called **/custom/custom.conf** and put new settings there. 
+     
+    **/conf/play.plugins** is used to enable or disable specific system functionality. To override these, create a file called **/custom/play.plugins**.
+
+4. Create a user account  
+
+    In a fresh installation Clowder is not configured with an email server, so when someone registers for an account the confirmation email will not be sent correctly. However the Clowder console will still display the contents of the email, so the confirmation URL can be copied from there. 
+
+A complete overview of installing Clowder is available [here](https://opensource.ncsa.illinois.edu/projects/artifacts/CATS/0.9.1/documentation/manual/installation.html).
+
+***
+
+## Interface
+Clowder consists of Datasets, Collections and Spaces.
+* **Datasets** consist of one or more files with associated metadata.
+* **Collections** consist of one or more datasets.
+* **Spaces** consist of collections and datasets. Spaces allow for particular roles to be assigned to particular users.
+
+There is also a search interface that supports querying by metadata or tags.
+
+In development is the ability to launch an Analysis Environment with a dataset attached to facilitate research.
+
+## API
+Clowder also includes a RESTful API that allows programmatic interactions such as creating new datasets and downloading files. For example, one can request a list of datasets using:
+    GET _clowder home URL_/api/datasets
+The current API schema for a Clowder instance can be accessed by selecting **API** from the **?** Help menu in the upper-right corner of the application.
 
-# Transferring data into Clowder using API
 Two example sources that will be pushing high data volumes into Clowder:
 * LemnaTec indoor system at Danforth (running)
 * LemnaTec outdoor system at Maricopa (in progress)
 
-## File delivery 
-Each file has some metadata associated with it. Generally, sources will write scripts against the Clowder API to post files to the database. Metadata can potentially be associated a few different ways. A generic workflow:
+**Scripting against the API**  
+For typical workflows, the following steps are sufficient to push data into Clowder in an organized fashion:
+
+1. Create a collection to hold relevant datasets (optional)
+
+    **POST /api/collections** _provide a name; returns collection ID_  
+    
+2. Create a dataset to hold relevant files and add it to the collection
+
+    **POST /api/datasets/createempty** _provide a name; returns dataset ID_  
+    **POST /api/collections/_collection id_/datasets/_dataset id_**  
+    
+3. Upload files and metadata to dataset
```

Comment:
Yes, we'll likely want a separate section dedicated to specific examples - in this portion, I'm just speaking broadly about the generic approach.